manifests/mod/passenger.pp

```diff
@@ -837,7 +837,11 @@
 
   if $::osfamily == 'RedHat' and $manage_repo {
     if $::operatingsystem == 'Amazon' {
-      $baseurl = 'https://oss-binaries.phusionpassenger.com/yum/passenger/el/6Server/$basearch'
+      if $::operatingsystemmajrelease == '2' {
+        $baseurl = 'https://oss-binaries.phusionpassenger.com/yum/passenger/el/7Server/$basearch'
```

Comment:
This URL doesn't exist. See https://oss-binaries.phusionpassenger.com/yum/passenger/el/
```suggestion
        $baseurl = 'https://oss-binaries.phusionpassenger.com/yum/passenger/el/7/$basearch'
```

Reply:
yeah. sorry for that. `6Server` directory also doesn't exist. fixed